### PR TITLE
🐛 Fix centered sizing and add max height defaults for text

### PR DESCRIPTION
**Pull request by chuckcarpenter**
closes: #837 

mobile:
image

and desktop:
image

### CODE REVIEW

File: src/js/components/shepherd-element.svelte [hunk 1]
```diff
@@ -120,6 +120,7 @@
     opacity: 0;
     outline: none;
     transition: opacity 0.3s;
+    width: 400px;
```
Comment (chuckcarpenter) [inline src/js/components/shepherd-element.svelte:123]: This fixes this issue in mobile
image

Comment (rwwagner90) [reply src/js/components/shepherd-element.svelte:123]: Hmm, I'm not sure if we want both `width` and `max-width`. Perhaps it should be `width:100%` with `max-width: 400px`?

File: src/js/components/shepherd-text.svelte [hunk 2]
```diff
@@ -24,6 +24,8 @@
     color: rgba(0, 0, 0, 0.75);
     font-size: 1rem;
     line-height: 1.3em;
+    max-height: 30vh;
```
Comment (rwwagner90) [inline src/js/components/shepherd-text.svelte:27]: Should we make this different for desktop/mobile or do you think this adequately works for most cases? I think I might prefer a max height of something like `75%`.

Comment (chuckcarpenter) [reply src/js/components/shepherd-text.svelte:27]: it kind of looks weird on desktop, with the max width of 400px, but I guess that's subjective.

Comment (chuckcarpenter) [reply src/js/components/shepherd-text.svelte:27]: also, it covers the element being pointed to, e.g.,

image

**Pull request merged**